### DISCUSSION TITLE
带宽的按量计费和恶意流量攻击防护

### DISCUSSION THREAD

**Discussion by xiguazhiPrince** [zeabur/.github#3] [Ideas]
在考虑带宽按量计费的时候，可以考虑下恶意攻击导致用户费用损失的情况吗？
不是每个开发者会考虑到这个问题和会开发安全相关的防御机制，那么开发平台是否可以提供防御机制，不需要开发者做额外开发和配置，这样可以帮助减少用户的损失。

**Comment by yuaanlin** [comment]
谢谢你的反馈！我们目前正在调研如何通过 CloudFlare 来帮助 Zeabur 的用户们可以更加快速且安全的访问自己的服务。
